README.md

```diff
@@ -1,129 +1,133 @@
+
+4. **GitHub Secrets**: Add any tokens or variables required for the plugin to GitHub secrets. This ensures sensitive information is securely stored. Refer to [GitHub secrets documentation](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#creating-secrets-for-a-repository) for guidance on creating secrets.
+
+5. **Enable the Plugin**: Simply set `enabled: true` to enable the new social media platform. This ensures that it will be implemented when creating posts.
+
+6. **Update publish_content.yml**: Finally, update the `publish_content.yml` file to include an environment variable referencing the token saved in GitHub secrets. Use the following template: (Don't put the prefixed `$` in here)
+
+```yaml
+...
+env:
+  TOKEN_SAVED_IN_PUBLISH_CONTENT: ${{ secrets.TOKEN_SAVED_IN_GITHUB_SECRETS }}
+...
+```
+
+Replace `TOKEN_SAVED_IN_GITHUB_SECRETS` with the name of the secret containing the token for the new social media platform.
+
+By following these steps, you can seamlessly integrate a new social media platform into Galaxy Social, expanding its reach and functionality.
+
+### Duplicate a Social Media Platform with Different Token
+
+If you need to use the same social media platform with different authentication tokens, you can duplicate the entry in the `plugins.yml` file. Follow these steps:
+
+1. **Duplicate Entry**: Copy the entry for the social media platform in the `plugins.yml` file and paste it below the original entry.
+
+2. **Update Name and Tokens**: Change the name of the duplicated entry to reflect the new configuration, and replace the token with the new authentication token.
+
+3. **Configuration**: Adjust any other configuration parameters as needed for the duplicated entry.
+
+4. **Use Name in Post**: Remember that the name you specify in the `plugins.yml` file must also be used within the `media` tag when creating a post. Ensure consistency to link the post with the correct social media platform.
+
+By following these steps, you can effectively duplicate a social media platform with a different token for specific use cases or configurations.
+
+## Run locally
+You can execute this repository on your machine by running `lib/galaxy_social.py` with the argument `--files Files ...` or `--folder FOLDER` to process files, or add `--preview` to preview the file as markdown. Also there is `--json-out processed_files.json` that could be change where to save the json results output.
+
+Remember to add the env variable that needed for each social media seperatly.
+
+## Social media implemented
+- Bluesky
+- Mastodon
+- Matrix: hashtags are included in the post but have no special function in matrix.
```

Comment:
and here's one more since the matrix plugin has been improved a bit since the last version.